### PR TITLE
refactor: make use of Command

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

While doing https://github.com/cowprotocol/cowswap/pull/3846 I had to add a lot of `(() => void) | null` all over the place.

I think is not super readable, then I remembered the Explorer had the type `Command` which is `() => void`, so I just add replaced all over the project every `() => void` to make use of `Command` type

So `(() => void) | null` becomes `Command | null`

😃 Not sure if you will like this change, just let me know if you are in favour of the change or not

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]
Nice!
Only one nitpick: probably we need to create another one library `@cowprotocol/types` and move common types there

**Comment by alfetopito** [comment]
> > Nice!
> > Only one nitpick: probably we need to create another one library @cowprotocol/types and move common types there
> 
> I 100% agree. I actually though about this, but felt lazy to do it (even this change I felt bad for this small detour)

Seems to be needed indeed, as the explorer build is not happy about the import the way it is 
image

**Review by alfetopito — APPROVED** [review]
Approved with the caveat there's one place in the explorer where the import needs to be updated (reason for the broken build)

File: libs/types/src/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export type Command = () => void
```
Comment (alfetopito) [inline libs/types/src/index.ts:1]: I thought you would move all the common types here, not just this one

**Pull request merged**